src/ai/backend/manager/services/image/actions/alias_image.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import Optional, override
+from uuid import UUID
+
+from ai.backend.manager.actions.action import BaseActionResult
+from ai.backend.manager.actions.exceptions import BaseActionException
+from ai.backend.manager.data.image.types import ImageAliasData
+from ai.backend.manager.services.image.base import ImageAction
+
+
+@dataclass
+class AliasImageAction(ImageAction):
+    image_canonical: str
+    architecture: str
+    alias: str
+
+    @override
+    def entity_id(self) -> Optional[str]:
+        return None
+
+    @override
+    def operation_type(self):
+        return "alias"
+
+
+@dataclass
+class AliasImageActionResult(BaseActionResult):
+    image_id: UUID
+    image_alias: ImageAliasData
+
+    @override
+    def entity_id(self) -> Optional[str]:
+        return str(self.image_id)
+
+    # TODO: Find a way to compare image alias row ID
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, AliasImageActionResult):
+            return False
+        return self.image_alias.alias == other.image_alias.alias
```

Comment:
Why do you need to define `__eq__`?

Reply:
Currently, in the AliasImageAction test, we cannot know the id of the `AliasImageRow` that will be created (`AliasImageActionResult.image_alias.id`).

Therefore, I overrode the `__eq__` to exclude this value from the comparison.

It would be ideal if there were a better way to pass the test without overriding the `__eq__` method.

Reply:
I don't want to add any real working code for test code.

Reply:
Okay, then I will remove the parts that are only necessary for testing purposes.